src/py42/_internal/initialization.py

```diff
@@ -40,7 +45,7 @@ def __init__(self, host_address, session_factory, root_session):
         )
 
         microservice_client_factory = MicroserviceClientFactory(
-            host_address, root_session, session_factory, self.user_context
+            host_address, root_session, session_factory, self.user_client, self.user_context
```

Comment:
Passing user_context and user_client in the wrong order.

Maybe we should be passing in key value arguments here